apps/src/templates/HideToolbarHelper.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import msg from '@cdo/locale';
+import cookies from 'js-cookie';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import trackEvent from '../util/trackEvent';
+import _ from 'lodash';
+
+const styles = {
+  closeX: {
+    position: 'absolute',
+    top: 5,
+    right: 5,
+    fontSize: 13
+  }
+};
+
+// Note that additional styling can be found in apps/style/HideToolbarHelper.scss.
+
+const HidetoolbarHelperCookieName = 'hide_toolbar_helper';
+
+/**
+ * An overlay with instrutions on hiding the toolbar for iPhone with iOS 13.
+ */
+export default class HideToolbarHelper extends React.Component {
+  state = {
+    showHelper: false
+  };
+
+  constructor(props) {
+    super(props);
+
+    // Track whether we knew the toolbar was shown last update, so that
+    // if it disappears, we can track the event in analytics...
+    this.wasToolbarShowing = false;
+
+    // ...but only track it once.
+    this.trackedToolbarHide = false;
+  }
+
+  updateLayout = () => {
+    const isiOS13 = navigator.userAgent.indexOf('iPhone OS 13') !== -1;
```

Comment:
It would be nice to start pulling these into util methods. i.e. so we could use something like `mobile.isiOS13()` here. Especially relevant for userAgent queries which seem to change somewhat frequently.

Reply:
That's a good point.  I'd like to factor out once we identity a second use-case.